### ISSUE TITLE
Some of the xhtml docs declare the html namespace and others don't.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6378]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)| @wsanchez reported|
|-|-|
|Trac ID|trac#6378|
|Type|enhancement|
|Created|2013-03-18 20:51:03Z|
|Branch|https://github.com/twisted/twisted/tree/htmldoc-namespace-6378|

Some of the xhtml docs declare the html namespace and others don't.  The one that do look like HTML documents when opened (from the sources) in Safari.  The others looks like XML documents, which are rendered as an XML hierarchy with disclosure triangles and whatnot.

Small thing, I guess, but we should be consistent, probably by adding the namespace declaration:

```
-<html>
+<html xmlns="http://www.w3.org/1999/xhtml">
```

<details><summary>Searchable metadata</summary>

```
trac-id__6378 6378
type__enhancement enhancement
reporter__wsanchez wsanchez
priority__lowest lowest
milestone__ 
branch__branches_htmldoc_namespace_6378 branches/htmldoc-namespace-6378
branch_author__wsanchez wsanchez
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1363639863000000 1363639863000000
changetime__1363760615000000 1363760615000000
version__None None
owner__wsanchez wsanchez

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_1"></a>|@wsanchez commented|
|-|-|

(In [37607]) Branch for [#6378](https://github.com/twisted/twisted/issues/6378)

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_2"></a>|@wsanchez commented|
|-|-|

It turns out that only a couple of files were missing this.  Several more were missing the DOCTYPE declaration, so I added those also.  Also <?xml> turds.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_4"></a>|@glyph set owner to @wsanchez|
|-|-|

Please revert the change in glossary.xhtml (it's in the middle, not at the top), and force a build to make sure that nothing untoward happened, then merge.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_5"></a>|@wsanchez set status to `closed`|
|-|-|

(In [37740]) Merge branches/htmldoc-namespace-6378: Add HTML and DTD declarations to docs.

Author: wsanchez
Reviewer: glyph
Fixes: [#6378](https://github.com/twisted/twisted/issues/6378)

Some docs lacked HTML and DTD declarations, this makes then consistantly present.
